fileio/private/read_nihonkohden_hdr.m

```diff
@@ -0,0 +1,105 @@
+function [hdr] = read_nihonkohden_hdr(filename)

```

Comment:
Why is this function part of the PR? Is this function referenced in ft_read_header and or ft_read_data?

Reply:
Sorry, that is from my last PR.....